support-frontend/assets/helpers/subscriptionsForms/validation.ts

```diff
@@ -69,6 +86,18 @@ function validate<Err>(rules: Array<Rule<Err>>): Err[] {
 	);
 }
 
+function deliveryAgentHasBeenChosen(
+	deliveryAgent: DeliveryAgentState,
+): boolean {
+	return deliveryAgent.chosenAgent ? true : false;
+}
+
+function deliveryAgentsAreAvailable(
+	deliveryAgent: DeliveryAgentState,
+): boolean {
+	return (deliveryAgent.response?.agents?.length ?? 0) > 0;
+}
```

Comment:
I see this function is now defined in two places: does that matter?

Reply:
I wasn't sure where the best place to share it would be so I thought a little duplication was better (it's a relatively simple function)

Reply:
Yeah, sounds good

Reply:
I might suggest defining once and importing